### DISCUSSION TITLE
A bright future

### DISCUSSION THREAD

**Discussion by BenVB1** [deckerst/aves#548] [Ideas]
Well, after many years using F-stop as a mainstay, something pops up which is even better. Well done!

I would like to suggest a brightness gesture for images, similar to the video section, as full brightness can be a little too bright.

**Comment by deckerst** [comment]
Haha, clever title, and I'm glad you enjoy the app!

As for the brightness gesture on images, I'm unlikely to implement it. Usage of images and videos are not really the same. For images, users often need to pan/zoom, which is rarely the case for videos, but the gestures for pan/zoom are incompatible with gestures for tweaking the brightness. It's a trade-off that makes sense for videos, but I don't think it does for images.

**Reply by BenVB1** [reply]
I was thinking along the lines of a small invisible area at the screen edge. Neither would clash then. Worth thinking about as it is a useful feature. But, up to you of course.
